lib/loader.js

```diff
@@ -17,26 +16,26 @@ module.exports = function loader(content) {
 
   const done = this.async();
   const loaderContext = this;
-  const { resourcePath, resource, context, loaderIndex } = loaderContext;
+  const { resourcePath, context, loaderIndex } = loaderContext;
```

Comment:
`context` was renamed to `rootContext` in `webpack >= 4` and needs to be checked via

```js
const context = loaderContext.rootContext || loaderContext.context || ...
```

Reply:
Hm, `this.context` and `this.rootContext` is not the same. `this.context` is the full path to the icons beeing collected, while `this.rootContext` is the path of the project using `svg-sprite-loader`.

Reply:
But it seems to me that it doesn't  matter, due to `stringifyRequest` using `path.relative` and node's way to lookup imports, to get the correct import.